src/app/(reskin)/history/components/history-table.tsx

```diff
@@ -69,16 +70,11 @@ function DesktopStatus({ type, data }: HistoryItem) {
   });
 
   return (
-    <TableCell className="dark:text-midGray hidden md:table-cell uppercase ">
-      {isConfirmed ? (
-        timestamp ? (
-          timestamp
-        ) : (
-          <Skeleton className="w-4 h-2" />
-        )
-      ) : (
-        "Pending"
-      )}
+    <TableCell className=" dark:text-midGray hidden md:table-cell">
+      <div className={timestamp ? "pt-[1lh]" : ""}>
```

Comment:
nit/qq: do we want the padding here? I find the version with the padding (pushing the timestamp on its own line) wasting space. With vs without padding:
<img width="998" height="312" alt="image" src="https://github.com/user-attachments/assets/8ef7e0cf-0066-4783-837e-51ce3ce00ab0" />
<img width="988" height="279" alt="image" src="https://github.com/user-attachments/assets/79c5438a-9729-4e74-b477-e966551ad3f0" />